src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -877,6 +919,29 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
     ) {
         const axisSelection = vertical ? this.vertSelection : this.horzSelection;
         const dataTestWhichAxis = vertical ? "Vertical" : "Horizontal";
+
+        let dataSourceLabel = "Profile";
+        let dataSourceValue = axisSelection.dataSourceId;
+        let dataSourceOptions = dataSourceOptionsByType[axisSelection.dataType+""] || [];
```

Comment:
how come you use this +"" instead of toString()?

Reply:
oh i dont know... for some reason thats where my head always goes. I can use toString instead

Reply:
oh now I remember, thats a pretty sloppy way of coercing undefined to index type lol.. Ill add a ternary check instead